installer/README.md

```diff
@@ -0,0 +1,70 @@
+# deno_install
```

Comment:
Let's use the name "installer" throughout to distinguish it from https://github.com/denoland/deno_install

Reply:
Ah, good catch, you're right!